.gitignore

```diff
@@ -17,4 +18,10 @@ dev_env.sh
 package-lock.json
 MANIFEST
 *.ipynb
-.pgpass
\ No newline at end of file
+.pgpass
+
+.idea/
+target/
+.bsp/
+create-job.json
+submit-run.json
```

Comment:
are this kind of objects created all around the project in different locations os just in a specific folder? Maybe it can be contained in another .gitignore inside the folder.

Reply:
Good idea :)